bee-api/bee-rest-api/src/types/responses.rs

```diff
@@ -14,33 +14,69 @@ use serde::{Deserialize, Serialize};
 pub struct InfoResponse {
     pub name: String,
     pub version: String,
+    pub status: StatusResponse,
+    pub protocol: ProtocolResponse,
+    pub metrics: MetricsResponse,
+    pub features: Vec<String>,
+    pub plugins: Vec<String>,
+}
+
+impl BodyInner for InfoResponse {}
+
+/// Returned in InfoResponse.
+/// Status information about the node.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct StatusResponse {
     #[serde(rename = "isHealthy")]
     pub is_healthy: bool,
-    #[serde(rename = "networkId")]
-    pub network_id: String,
+    #[serde(rename = "latestMilestoneTimestamp")]
+    pub latest_milestone_timestamp: u64,
+    #[serde(rename = "latestMilestoneIndex")]
+    pub latest_milestone_index: u32,
+    #[serde(rename = "confirmedMilestoneIndex")]
+    pub confirmed_milestone_index: u32,
+    #[serde(rename = "pruningIndex")]
+    pub pruning_index: u32,
+}
+
+/// Returned in InfoResponse.
+/// Protocol information about the node.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct ProtocolResponse {
+    #[serde(rename = "networkName")]
+    pub network_name: String,
     #[serde(rename = "bech32HRP")]
     pub bech32_hrp: String,
     #[serde(rename = "minPoWScore")]
     pub min_pow_score: f64,
+    #[serde(rename = "rentStructure")]
+    pub rent_structure: RentStructureResponse,
+}
+
+/// Returned in InfoResponse.
+/// Rent information about the node.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RentStructureResponse {
+    #[serde(rename = "vByteCost")]
+    pub vbyte_cost: u64,
```

Comment:
If I'm not mistaken, `vbyte_cost` can be a floating point value.

Reply:
Hmm here it is u64 https://github.com/gohornet/hornet/blob/764e4ee25995b1557129277a97dea244b923513e/core/protocfg/core.go#L61

And in the RFC I could find the datatype https://github.com/muXxer/protocol-rfcs/blob/master/tips/TIP-0019/tip-0019.md

Are there any other resources?

Reply:
Hmm, they might have changed the TIPs again. In [this section](https://github.com/muXxer/protocol-rfcs/blob/master/tips/TIP-0019/tip-0019.md#weights-for-different-outputs) of the TIP they specify floating point values for the weights. Also, the equation for the byte cost specifies rounding down, so at least one of these parameters can end up being a float.

Reply:
https://github.com/Wollac/protocol-rfcs/blob/protocol-parameters/tips/TIP-0022/tip-0022.md here it's just `500`

Reply:
yes the question is which one. it makes more sense for the `byte_cost` to be a float, but doesn't neccessarly need to be the case I guess

Reply:
[This section](https://github.com/muXxer/protocol-rfcs/blob/master/tips/TIP-0019/tip-0019.md#weights-for-different-outputs) shows floating point values for the weights. Other than that, I agree there is not much information about this in the TIP. I've left a comment: https://github.com/iotaledger/tips/pull/39#discussion_r801416125

Reply:
So I guess we can let the PR unchanged as both values seem to be a integer.